BootloaderCommonPkg/Library/ContainerLib/ContainerLib.c

```diff
@@ -276,6 +276,26 @@ AuthenticateComponent (
   return Status;
 }
 
+/**
+  Return Containser Key Type based on its signature
+
+  @param[in]    ContainerSig    Signature of the container
+
+  @retval                       COMP_TYPE_PUBKEY_OS for CONTAINER boot image
+                                COMP_TYPE_PUBKEY_CFG_DATA, otherwise
+**/
+UINT8
+GetContainerKeyTypeBySig (
+  IN  UINT32    ContainerSig
+  )
+{
+  if (ContainerSig == SIGNATURE_32 ('B', 'O', 'O', 'T')) {
```

Comment:
Can we use BOOT_MAGIC_PATTERN here? A single signature definition would be better.